doc/subsystems/bluetooth/devel.rst

```diff
@@ -41,7 +41,7 @@ The host OS's Bluetooth controller is connected to the second QEMU
 serial line using a UNIX socket. This socket employs the QEMU option
 :literal:`-serial unix:/tmp/bt-server-bredr`. This option is already
 added to QEMU through :makevar:`QEMU_EXTRA_FLAGS` in most Bluetooth
-sample Makefiles' and made available through the 'run' make target.
+sample CMakeLists.txt files and made available through the 'run' make target.
```

Comment:
This part conflicts with https://github.com/zephyrproject-rtos/zephyr/pull/4893

How should we resolve the conflict?

Reply:
I perfer your version, so I'll remove this from the PR

Reply:
done